commands/command_filter_process.go

```diff
@@ -78,11 +132,19 @@ func filterCommand(cmd *cobra.Command, args []string) {
 			malformedOnWindows = append(malformedOnWindows, req.Header["pathname"])
 		}
 
+		if delayed {
+			w = nil
+		}
+
 		var status string
 		if ferr := w.Flush(); ferr != nil {
 			status = statusFromErr(ferr)
 		} else {
-			status = statusFromErr(err)
+			if delayed {
+				status = delayedStausFromErr(err)
+			} else {
+				status = statusFromErr(err)
+			}
```

Comment:
Way better -- thanks for this suggestion. Changed things up in 1001e1c (and I got that typo!)